orchestration/OrchestrationService/ModuleConfigurationDeployment.ps1

```diff
@@ -646,22 +712,22 @@ Function Get-AuditStorageInformation {
             LocalPath = ''
         };
 
-        if ($ArchetypeInstanceJson.ToolkitComponents.Audit -and
-        $ArchetypeInstanceJson.ToolkitComponents.Audit.StorageType.ToLower() -eq "storageaccount"){
+        if ($ToolkitConfigurationJson.Configuration.Audit -and
```

Comment:
Same as above, we should default to Local storage if no Audit setting exists or no file exists

Reply:
Updated.